README.md

```diff
@@ -27,3 +28,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+----
+
+As we discussed in class and how form the feedback from last semester: an appropriate fibonacci invariant is to check at the beginning of a recursive call
+if the main property of fibonacci is maintained. The main property is that the new number (n) will be equal to (n-1) + (n-2). In my code I have the base cases
```

Comment:
That's not an invariant. What do you need to be true for the function to do the right thing?